### ISSUE TITLE
[Suggestion] More streamlined way to switch elytra fly speeds (and similar speeds)

### ISSUE THREAD

**Issue by w014prc** [MatHax/Client#92]
### Describe the feature

Is it possible to make it so that you can have two separate elytra speeds, and switch back and forth between with a hotkey or something without changing configuration? This is one of the biggest time sucks for me; travelling on a long nether highway at 40 bps, and then going to the overworld and manually lowering it to like 30 or even 28 so that I can safely maneuver around a build, and then manually raising it back up to 40 if I have to go on the nether highway again.

I can make two identical configs with different wither speeds, but that is clunky and might turn some hacks on or off that I don't want if I set the configs a long time ago. Is it possible to make, like, sub-configs, specifically for elytra speed, or at least for sliders in general? (this is a problem that also manifests with speed and boat fly)

Basically, I think there should be an easier way to switch how fast elytra fly is, as it is something I do several times an hour

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client.
(I have checked every module and their settings on the **repository**)

- [X] This wasn't already suggested.
(I have checked every suggestion on github)

- [X] This feature is a valid suggestion.
(I have read the FAQs and Guides)
